superset-frontend/src/dashboard/components/RefreshIntervalModal.jsx

```diff
@@ -74,12 +102,32 @@ class RefreshIntervalModal extends React.PureComponent {
               value={this.state.refreshFrequency}
               onChange={this.handleFrequencyChange}
             />
+            {showRefreshWarning && (
+              <div className="refresh-warning-container">
+                <Alert bsStyle="warning">
+                  <div>{refreshWarning}</div>
+                  <br />
+                  <div className="bold">
```

Comment:
instead of this, why not use a `<strong>` tag instead? We wouldn't need the extra css class

Reply:
good catch. fixed.